api/pom.xml

```diff
@@ -70,6 +70,11 @@
             <artifactId>handlebars</artifactId>
         </dependency>
 
+		<dependency>
+			<groupId>org.graalvm.sdk</groupId>
```

Comment:
Shouldn't we instead be using `js-scriptengine` and `js` for with group `org.graalvm.js`?

Reply:
Yes, I didn't test this with Java 17 well. I'll update the pull request.